python/cudf/cudf/core/column/string.py

```diff
@@ -2014,11 +2014,47 @@ def get_json_object(self, json_path):
         -------
         Column: New strings column containing the retrieved json object strings
 
-        """
-        
-        return self._return_or_inplace(
-            cpp_get_json_object(self._column, cudf.Scalar(json_path, "str"))
+        Examples
+        --------
+        >>> import cudf
+        >>> s = cudf.Series(
+         [
+              \"\"\"{"store": {
+                           "book": [
+         { "category": "reference",
+             "author": "Nigel Rees",
+             "title": "Sayings of the Century",
+                          "price": 8.95
```

Comment:
alignment seems off here - maybe we can use something like https://jsonformatter.curiousconcept.com/ to format the JSON beforehand?

Reply:
Here's what I got after processing the JSON using that tool:

```json
{
   "store":{
      "book":[
         {
            "category":"reference",
            "author":"Nigel Rees",
            "title":"Sayings of the Century",
            "price":8.95
         },
         {
            "category":"fiction",
            "author":"Evelyn Waugh",
            "title":"Sword of Honour",
            "price":12.99
         }
      ]
   }
}
```